src/main.ts

```diff
@@ -527,7 +527,7 @@ export default class SRPlugin extends Plugin {
         ease = schedObj.ease;
 
         const due = window.moment(now + interval * 24 * 3600 * 1000);
-        const dueString: string = due.format("YYYY-MM-DD");
+        const dueString: string = due.format("YYYY-MM-DD-HH-mm");
```

Comment:
I would suggest that the time component is only included if necessary. This would keep the HTML commands in the current format, and only the longer format when needed.